pkg/onboarding/infrastructure/database/fb/firebase.go

```diff
@@ -4214,6 +4214,76 @@ func (fr *Repository) CheckIfRoleNameExists(ctx context.Context, name string) (b
 	return false, nil
 }
 
+// GetUserProfilesByRoleID returns a list of user profiles with a particular ID
```

Comment:
Make this comment descriptive. Particular ID is vague -->  Particular Role ID  makes more sense